CodeGeneration/Sources/Utils/CodeGenerationFormat.swift

```diff
@@ -36,7 +36,11 @@ public class CodeGenerationFormat: BasicFormat {
   }
 
   public override func visit(_ node: CodeBlockItemSyntax) -> CodeBlockItemSyntax {
-    if node.parent?.parent?.is(SourceFileSyntax.self) == true, !node.item.is(ImportDeclSyntax.self) {
+    // First item in the `CodeBlockItemListSyntax` don't need a newline or identation if the parent is a `SourceFileSyntax`.
+    // We want to group imports so newline between them should be omitted
+    let shouldOmitNewline = node.parent?.as(CodeBlockItemListSyntax.self)?.first == node || node.item.is(ImportDeclSyntax.self)
```

Comment:
We could also add this for `EnumCaseDeclSyntax`

So 
```swift
enum Foo {
case bar

case baz
}
```

becomes

```swift
enum Foo {
case bar
case baz
}
```

Reply:
Sounds reasonable to me. To make sure the expression doesn’t get too large, it might make sense to refactor it to a separate function, and also move the `node.parent?.parent?.is(SourceFileSyntax.self)` check to that function.

Also, I think `shouldOmitNewline` is not really the right name here because there will still be a newline before the `import`, just not two. So the function should probably be named `shouldBeSeparatedByTwoNewlines` or something like that.